README.md

```diff
@@ -97,11 +97,11 @@ Scarpe allows you to modify the app's behaviour outside of the normal Shoes API
 
 For example, we are working with multiple display services like Webview, Glimmer, and possibly some others.
 
-The SCARPE_DISPLAY_SERVICES environment variable allows you to choose one or more display services, from the default Webview service, to no service at all, to potentially other experimental or incomplete services. This may be important if you're developing a new display method for Scarpe. Normally ScarpeDisplayServices will contain a semicolon-delimited list of class names for display services (which can be just the name of a single display service). For no display service at all, set it to a single dash.
+The SCARPE_DISPLAY_SERVICE environment variable allows you to choose one or more display services, from the default Webview service to potentially other experimental or incomplete services. This may be important if you're developing a new display method for Scarpe. The display service variable will contain a name like "wv_local" or "wv_remote" or "glibui" which corresponds to a require-able Ruby file under lib/scarpe, either in the Scarpe gem or another gem your app requires.
 
 Example usage:
 
-`SCARPE_DISPLAY_SERVICES=Scarpe::GlimmerLibUIDisplayService ./exe/scarpe examples/hello_world.rb`
+`SCARPE_DISPLAY_SERVICE=glibui ./exe/scarpe examples/hello_world.rb`
```

Comment:
I didn't test out glimmerlibui manually locally but this will be a lot nicer approach to have

Reply:
Yeah, I feel like the old way with the whole Ruby constant was kind of an embarrassment, but I didn't initially know what to do instead. I think this way is much better.